doc/en/user/source/community/features-templating/directives.rst

```diff
@@ -1070,7 +1070,17 @@ Regarding the :code:`$includeFlat` option is worth mentioning that in a JSON con
 * If a JSON object is included, then its properties are directly included in-place, which makes sense only within another object. 
 * If instead a JSON array is included, then its values are directly included in-place, which makes sense only within another array.
 
-The following JSON snippet shows the four possible syntax options for template inclusion:
+The following JSON snippet shows the five possible syntax options for template inclusion:
+
+${property}:
+
+.. code-block:: json
+   :linenos: 
+
+    {
+       "aProperty": "10", 
+       "bProperty": "20"
+    }
```

Comment:
Looks a bit confusing. I would leave this pieces of documentation as it is with the four example that explains the four possible syntax of the include directive then I would a paragraph below explaining that in case of an includeFlat directive  is specified and its attribute value is a property interpolation directive, if the property name evaluates to a json it gets included flat in the final output.

Reply:
done